common/core/gpudevice.cpp

```diff
@@ -201,6 +201,8 @@ void GpuDevice::DisplayManager::HotPlugEventHandler() {
           "UpdateDisplayState.");
       UpdateDisplayState();
     }
+    /* Adding sleep to avoid busy looping before a retry */
+    usleep(1000);
```

Comment:
ya, probably we can split this sleep thing from this PR. In subsequent one, move to make use of lseek. I am afraid we will run into nasty race issues with sleep when u have "Real" hot plug events. Is that fine @yogesh-marathe